x/bank/module.go

```diff
@@ -81,6 +81,7 @@ func (AppModuleBasic) RegisterGRPCGatewayRoutes(clientCtx client.Context, mux *g
 	if err := types.RegisterQueryHandlerClient(context.Background(), mux, types.NewQueryClient(clientCtx)); err != nil {
 		panic(err)
 	}
+
 }
 
 // GetTxCmd returns the root tx command for the bank module.
```

Comment:
Something which also affects other modules: in `heimdall-v1` we used to have
 `_ hmModule.HeimdallModuleBasic = AppModule{}`.
Do you think this is needed?

Reply:
Same goes for `VerifyGenesis` (which anyway returns `nil`)

Reply:
Also, `contractCaller` is definitely not used in `heimdall-v1` `bank` module, right? I didn't see any import here.

Reply:
`HeimdallModuleBasic` was from legacy I think. `ValidateGenesis` is the replacement for `VerifyGenesis` (which doesn't return nil though).